src/components/icon-button/icon-button.scss

```diff
@@ -33,6 +37,10 @@
 
     limel-icon {
         @include mixins.is-elevated-clickable;
+        background-color: var(
+            --specified-background-color,
+            rgb(var(--contrast-100))
+        );
```

Comment:
I think a better default value is `rgb(var(--contrast-100), 0.8)` or something like that. This way, if the icon-button is used on a colored background (let's say a header), it'll still blend in quite well. Because some of the background color will leak through the button, and the consumer doesn't have to necessarily change the icon-button's background color.